### PR TITLE
Snippet_Table module

### PR DESCRIPTION
**Pull request by EugenPetrov**

### CODE REVIEW

File: src/Snippet/Table/Controller/Index/Create.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+     */
+    public function execute()
+    {
+        $data = [
+            'name' => 'Table',
+            'description' => 'table creation and crud operations examples',
+        ];
+        $this->snippetModel->addData($data);
+        $this->snippetModel->getResource()->save($this->snippetModel);
+        var_dump('Inserted: ', [
```
Comment (DmitryFurs) [inline src/Snippet/Table/Controller/Index/Create.php]: Use ```$this->getResponse()->appendBody();```

File: src/Snippet/Table/Controller/Index/Read.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+    /**
+     * @return void
+     */
+    public function execute()
+    {
+        $collection = $this->snippetModel->getCollection();
+
+        foreach ($collection as $item) {
+            var_dump('Item ID: ' . $item->getEntityId());
```
Comment (DmitryFurs) [inline src/Snippet/Table/Controller/Index/Read.php]: Use ```$this->getResponse()->appendBody();```

File: src/Snippet/Table/Controller/Index/Read.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            var_dump($item->getData());
```
Comment (DmitryFurs) [inline src/Snippet/Table/Controller/Index/Read.php]: Use ```$this->getResponse()->appendBody();```

File: src/Snippet/Table/Setup/UpgradeData.php [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    }
+
+    /**
+     * @param ModuleDataSetupInterface $setup
+     * @param ModuleContextInterface $context
+     * @return $this
+     */
+    private function upgrade101(ModuleDataSetupInterface $setup, ModuleContextInterface $context)
+    {
+        if (version_compare($context->getVersion(), '1.0.1') >= 0) {
```
Comment (DmitryFurs) [inline src/Snippet/Table/Setup/UpgradeData.php]: Use third parameter ```operator```, then function will return ```boolean```
http://php.net/manual/en/function.version-compare.php#refsect1-function.version-compare-parameters

File: src/Snippet/Table/Controller/Index/Create.php (hunk 1 above)
Comment (EugenPetrov) [inline src/Snippet/Table/Controller/Index/Create.php]: Done

File: src/Snippet/Table/Controller/Index/Read.php (hunk 2 above)
Comment (EugenPetrov) [inline src/Snippet/Table/Controller/Index/Read.php]: Done

File: src/Snippet/Table/Controller/Index/Read.php (hunk 3 above)
Comment (EugenPetrov) [inline src/Snippet/Table/Controller/Index/Read.php]: Done

File: src/Snippet/Table/Setup/UpgradeData.php (hunk 4 above)
Comment (EugenPetrov) [inline src/Snippet/Table/Setup/UpgradeData.php]: Done

**Pull request merged**